location/gql_queries.py

```diff
@@ -29,7 +29,7 @@ def resolve_client_mutation_id(self, info):
 
     @classmethod
     def get_queryset(cls, queryset, info):
-        return Location.get_queryset(queryset, info)
+        return Location.get_queryset(queryset, info.context.user)
```

Comment:
The function does
```
        if isinstance(user, ResolveInfo):
            user = user.context.user
```
I kept the full resolveinfo because it contains context info that might be useful sometimes and we can let the get_queryset decide.
The fact that we support both is because it's used by the FHIR API that has a user directly and GraphQL that has resolveinfo.